File: example-java/src/main/java/com/github/simonpercic/oklogexamplejava/data/api/okhttp/RestApi.java
package com.github.simonpercic.oklogexamplejava.data.api.okhttp;

import com.github.simonpercic.oklogexamplejava.data.api.model.response.show.ShowResponse;

import java.util.List;

import retrofit.http.GET;

/**
 * @author Simon Percic <a href="https://github.com/simonpercic">https://github.com/simonpercic</a>
 */
public interface RestApi {

    @GET("/shows")
    List<ShowResponse> getShows();
}

File: example-kotlin/src/main/kotlin/com/github/simonpercic/oklogexamplekotlin/data/api/okhttp/RestApi.kt
package com.github.simonpercic.oklogexamplekotlin.data.api.okhttp

import com.github.simonpercic.oklogexamplekotlin.data.api.model.response.show.ShowResponse

import retrofit.http.GET

/**
 * @author Simon Percic [https://github.com/simonpercic](https://github.com/simonpercic)
 */
interface RestApi {

    @GET("/shows")
    fun getShows(): List<ShowResponse>
}
